src/lib/Controls.scss

```diff
@@ -1,15 +1,6 @@
-.bp-controls-wrapper {
+.bp-controls {
     position: absolute;
     bottom: 25px;
-    left: 50%;
```

Comment:
It doesn't seem like the control bar will be centered on the screen. Do we need to do something like `left: 50%; transform: translateX(-50%);`?

Reply:
> It doesn't seem like the control bar will be centered on the screen. Do we need to do something like `left: 50%; transform: translateX(-50%);`?

I did it in the parent container, which you think it's risky below